src/Database/Schema/PostgresSchema.php

```diff
@@ -334,7 +334,10 @@ public function columnSql(Table $table, $name)
             'binary' => ' BYTEA',
             'float' => ' FLOAT',
             'decimal' => ' DECIMAL',
+            'tinytext' => ' TEXT',
```

Comment:
I would have expected `TEXT`. The various TEXT columns in MySQL are a :snowflake: when compared to what other DB's do.